KsApi/models/graphql/adapters/Project+ProjectFragment.swift

```diff
@@ -228,12 +228,14 @@ private func extendedProject(from projectFragment: GraphAPI.ProjectFragment) ->
   let risks = projectFragment.risks
   let environmentalCommitments = extendedProjectEnvironmentalCommitments(from: projectFragment)
   let faqs = extendedProjectFAQs(from: projectFragment)
+  let minimumSingleTierPledgeAmount = projectFragment.minPledge
```

Comment:
What does single tier mean here?

Reply:
single tier is just a reward basically, just a fancy name for single `reward` (got it from the GQL Schema Docs for `fxRate` - `The min pledge amount for a single reward tier.`)

Reply:
Want to change it to `minimumRewardPledgeAmount`?

Reply:
Nope, as long as it references something I think it's fine. I wasn't aware there were tiers until just now.